tests/functional/test_config_groups_data/cluster_with_group_all_params/cluster/config.yaml

```diff
@@ -0,0 +1,284 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- type: cluster
+  name: test_cluster
+  version: 1.5
+  edition: community
+  config_group_customization: true
```

Comment:
it option should be tested with a different test cases
probably you can create a second bundle, and use parametrization on current test cases

Reply:
yes, I want to add test cases with false values in the next pull request, this one is already big enough to review